SUMMARY.md

```diff
@@ -0,0 +1,19 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > In this lesson, you will learn how to simplify the condition of if/else statements using ternary operator. You will also learn the advantages of using ternary operator as an expression.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to identify the expected data type being passed as arguments in a function declaration. 
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to start a PHP session, get PHP session variable values, modify a PHP session variable, and destroy a PHP session.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create/retrieve cookies, modify cookie values, delete cookies, and check if cookies are enabled.
+
+* [Exceptions](content/Exceptions/Exceptions.q.md) ** - minutes**
+    > In this lesson, you will learn the basic usage of exceptions and create custom exceptions that inherits the `Exception` class using the `extends` keyword.
+
+* [Error Handling](content/ErrorHandling/ErrorHandling.q.md) ** - minutes**
+    > In this lesson, you will learn how to use exception handling keywords such as `try, throw, catch, and finally` to handle an exception.
```

Comment:
I think we need to replace this statement here... 

Error Handling - is a way of proper dealing with errors encountered when running a program. It is a way of handling and logging the errors properly. In here we should not introduce `try` and `catch` keywords instead we can use existing PHP Error handling functions like `error_log()`, `trigger_error()`, and etc. Please try to check more sources thanks.

Reply:
Suggested sequence:

- Error Handling

- Basic Exception - simple `try...catch` with basic exception classes like `RuntimeException`, `OutOfBoundsException` and etc. Specifying the hierarchy importance.

- Exception.... - This is where we introduce on how to throw an exception.

- Custom Exception - This is where we teach learners on how to create their own exception.

Please try to review this one and read further. Thanks.

Reply:
Please try to also check if which part of the course can we introduce `Serialization` lesson. I think we need to include this in the Course. Thanks.